dace/frontend/fortran/ast_transforms.py

```diff
@@ -231,7 +1025,7 @@ def visit_Call_Expr_Node(self, node: ast_internal_classes.Call_Expr_Node):
                 "malloc", "pow", "cbrt", "__dace_epsilon", *FortranIntrinsics.call_extraction_exemptions()
```

Comment:
is malloc a proper fortran intrinsic?

Reply:
Good catch, it's a remnant that should be removed